src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -184,6 +223,9 @@ def get_profile(
         self,
         profile_name: Optional[str] = None,
         profile_type: Optional[str] = None,
+        config_type: Optional[str] = None,
+        validate_schema: Optional[bool] = False,
```

Comment:
I think the default value for `validate_schema` should be True (same as above) 😋